.travis.yml

```diff
@@ -33,7 +33,9 @@ before_install:
   - if [[ $COVERAGE != yes ]]; then phpenv config-rm xdebug.ini || true; fi;
 
 install:
-  - composer update --prefer-dist --no-interaction
+  - export COMPOSER_FLAGS="--prefer-dist --no-interaction"
+  - if [[ $TRAVIS_PHP_VERSION = nightly ]]; then export COMPOSER_FLAGS="$COMOPSER_FLAGS --ignore-platform-reqs"; fi
```

Comment:
`--ignore-platform-reqs` is needed because both our dependencies, and this project itself don't allow php 8.0 yet.

I think its not good to declare us to be compatible with php 8 just yet, as we have no idea if we are going to be.